models/prices/solana/prices_solana_tokens.sql

```diff
@@ -15,7 +15,7 @@ SELECT
 FROM
 (
     VALUES
-
+    ("hades","solana","HADES","BWXrrYFhT7bMHmNBFoQFWdsSgA3yXoAnMhDK6Fn1eSEn",9),
```

Comment:
this is not a valid id value on coinpaprika -- can you find the correct one? if you search on their site, usually the url will contain the id. i saw `hades-hades` but that is on ethereum and not active

Reply:
ah yeah was just checking

Reply:
okay I've added only tokens that are active on coinpaprika now!